common/messages/revealEntry.go

```diff
@@ -186,15 +191,20 @@ func (m *RevealEntryMsg) Validate(state interfaces.IState) int {
 		if eb == nil {
 			state.LogMessage("executeMsg", "Hold, no chain", m)
 			// No chain, we have to leave it be and maybe one will be made.
-			return 0
+			//old holding .., return 0
```

Comment:
If there is no chain, we still accept the commit.
So for the dependent holding, will this reveal ever get out? 

Should it be dependent on the chain instead of the entry hash? (Would need to keep a list of entries depending on a chain then)

Reply:
Nvm, it does do that